tests/test_translation_level_02.py

```diff
@@ -11,6 +13,7 @@
 
 class TestsTranslationLevel2(HedyTester):
     level = 2
+    all_kwords = hedy_translation.all_keywords_to_dict()
```

Comment:
Thanks for this work @racertop it is a great way to test more extensively!!

I would just like to ask, can we make this var name `all_keywords`? It is just 2 more letters and it is a lot more clear! In general we try not to use abbreviations unless they really save letters and are wellknown (like AST for abstract syntax tree)

Reply:
Ok! Do you want me to change it in the places where I use `print_kword` or `if_kword` too?

Reply:
Yeah that would be nicer!